cmd/init-repo/main.go

```diff
@@ -97,6 +98,31 @@ func main() {
 		glog.Fatalf("Failed to load rules: %v", err)
 	}
 
+	goVersions := []string{DefaultGoVersion}
+	for _, rule := range rules.Rules {
+		for _, branch := range rule.Branches {
+			if branch.GoVersion != "" {
+				found := false
+				for _, v := range goVersions {
+					if v == branch.GoVersion {
+						found = true
+					}
+				}
+				if !found {
+					goVersions = append(goVersions, branch.GoVersion)
+				}
+			}
+		}
+	}
+	for _, v := range goVersions {
+		installGoVersion(v, filepath.Join(SystemGoPath, "go-"+v))
+	}
+	goLink, target := filepath.Join(SystemGoPath, "go"), filepath.Join(SystemGoPath, "go-"+DefaultGoVersion)
+	os.Remove(goLink)
+	if err := os.Symlink(target, goLink); err != nil {
```

Comment:
@sttts  just to make sure: we are not supporting parallel build, am i right? 😃

Reply:
No, we don't. But even then, this is just the default Go if no version is specified.

The whole go-workspace directory is a scratch pad for the publishing bot. I.e. it is used exclusively by one bot.